src/config.ts

```diff
@@ -40,6 +40,18 @@ const ndlaApiUrl = () => {
   }
 };
 
+const ndlaFrontendUrl = () => {
+  switch (ndlaEnvironment) {
+    case 'local':
```

Comment:
Egentlig burde local være 'http://localhost:30017' eller 'http://localhost:3000' i og med at det er der ndla-frontend kjører om du kjører den lokalt. ndlaApiUrl er jo satt opp på samme måten.

Reply:
Ja, jeg er forsåvidt enig i det, men da vil man få problemer når oembed-proxy kalles, da den (så vidt jeg kan se) ikke har en provider for localhost og kaster 501 feil.  Men om vi vil ha det som localhost uansett så kan jeg raskt endre på det.